test/cases/BaseDeployMissionUnit.test.ts

```diff
@@ -0,0 +1,214 @@
+import assert from "node:assert";
+import {
+  DBVersioner,
+  DeployCampaign,
+  HardhatDeployer,
+  IContractState,
+  IDeployCampaignConfig,
+  IHardhatBase,
+  ISignerBase,
+  MongoDBAdapter,
+} from "../../src";
+import { HardhatMock } from "../mocks/hardhat";
+import { loggerMock } from "../mocks/logger";
+import { testMissions } from "../mocks/missions";
+import { MongoClientMock } from "../mocks/mongo";
+
+
+// this.timeout() doesn't work for arrow functions.
+describe("Base deploy mission", function () {
+  this.timeout(5000);
```

Comment:
why is this here?

Reply:
From time to time, flow falls by timeout of 2000ms. When I increased this, it helps. I know, its not for our soft, It's for my machine. But I just wanted to keep it while the PR has WIP status. Or should I remove it before each push?